prisma/migrations/20220205135022_add_verified_column/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `smartContractAddress` on the `EventType` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "EventType" DROP COLUMN IF EXISTS "smartContractAddress";
```

Comment:
Things like this should really be in a separate PR, because it's not clear that the removal of the `smartContractAddress` column would be found in the verified badge PR

Reply:
oh i dont even know where this came from. i really wanna get this merged, is this a hard requirement?

Reply:
added the title: added verified badge to profile and removed/dropped smartContractAddress from db #1710